public/app/features/admin/containers/ServerStats.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { PureComponent } from 'react';
```

Comment:
Suggest to move this and test file to public/app/features/admin/ServerStats(.test).tsx and __snapshots__ to  public/app/features/admin/__snapshots__ to be consistent with alerting feature.

Reply:
Yes, the file structure for admin has yet to be updated